pages.es/common/az-vm.md

```diff
@@ -0,0 +1,33 @@
+# az vm
+
+> Administra máquinas virtuales en Azure.
+> Parte de `az`, el cliente de línea de comandos para Microsoft Azure.
+> Más información: <https://learn.microsoft.com/cli/azure/vm>.
+
+- Lista los detalles de las máquinas virtuales disponibles:
+
+`az vm list`
+
+- Crea una máquina virtual `UbuntuServer 18.04 LTS` y genera claves ssh:
+
+`az vm create --resource-group {{grupo_de_recursos}} --name {{nombre}} --image {{Canonical:UbuntuServer:18.04-LTS:latest}} --admin-user {{usuario_azure}} --generate-ssh-keys`
```

Comment:
Based on the first of this list https://learn.microsoft.com/en-us/cli/azure/vm?view=azure-cli-latest#az-vm-create-examples we can update this example to be out of time. This should be updated on the English page too
```suggestion
`az vm create --resource-group {{grupo_de_recursos}} --name {{nombre}} --image {{UbuntuLTS}} --admin-user {{usuario_azure}} --generate-ssh-keys`
```